### PR TITLE
Fixing linting problems, style, and old problems

### PR DESCRIPTION
**Pull request by christopherhafke**
Big boy merge: most files are just linting/style.
Real changes to the mailing list form.

### CODE REVIEW

File: src/utility/geometryangle.min.js [hunk 1]
```diff
@@ -1,7 +1,1607 @@
-import $ from 'jquery';
+import $ from "jquery";
```
Comment (eashwar) [inline src/utility/geometryangle.min.js]: This should not be unminified.

File: src/App.css [hunk 2]
```diff
@@ -1,28 +1,32 @@
 .App {
-    text-align: center;
+	text-align: center;
```
Comment (eashwar) [inline src/App.css:2]: I generally prefer spaces to tabs, but this is not a big complaint as the codebase is consistent.

File: src/components/Team/CardContainer.css [hunk 3]
```diff
@@ -1,21 +1,19 @@
 .cards {
-    display: inline-flex;
-    flex-wrap: wrap;
-    justify-content: center;
-    width: 100%;
- }
+	display: inline-flex;
```
Comment (eashwar) [inline src/components/Team/CardContainer.css:2]: We should consider refactoring to use `styled-components`, eventually.

**Review by eashwar — CHANGES_REQUESTED** [review]
Only one 'real' change requested: (to re-minify geometry angles). Otherwise, LGTM.

**Review by eashwar — APPROVED** [review]

File: src/components/Team/CardContainer.css (hunk 3 above)
Comment (christopherhafke) [reply src/components/Team/CardContainer.css:2]: agreed

**Pull request merged**